### ISSUE TITLE
Marketprice high memory usage

### ISSUE THREAD

**Issue by ddengster** [viabtc/viabtc_exchange_server#189]
Hi, 
For marketprice, load_market() does a HGETALL for all values when it does load_market_kline. We have up to 290k values for one of our markets with key "k:marketname:1m", and as you can guess it can take a long time to load. Is it possible to specify a range for the HGETALL call?

**Comment by lawrence296** [comment]
You're right, load time can get long when data is huge. What I did was to purge "1m" data more than 3 months old.
